### PR TITLE
change import in src and test to support RXJS v6

### PR DESCRIPTION
**Pull request by kmathy**

### CODE REVIEW

**Comment by kmathy** [comment]
#10 related

File: package.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
   "devDependencies": {
     "@types/jest": "^22.0.1",
     "jest": "^22.0.5",
-    "rxjs": "^5.5.0",
+    "rxjs": "^6.0.0",
```
Comment (NetanelBasal) [inline package.json]: I think we can use 6.2.0

**Comment by NetanelBasal** [comment]
Great, thanks! I will release a new version tonight.

File: package.json (hunk 1 above)
Comment (kmathy) [reply package.json]: We can, but the feature is already ready since 6.0.0 I think, but it doesn't matter :)

**Comment by NetanelBasal** [comment]
3.0.0, thanks!

**Pull request closed without merging**